admin/acceptance.py

```diff
@@ -194,12 +200,31 @@ def start_nodes(self, reactor):
 
         for node in self.NODE_ADDRESSES:
             yield remove_known_host(reactor, node)
-        returnValue([
+
+        certificates_path = FilePath(mkdtemp())
```

Comment:
This seems like it should be deleted in this branch, along with `configure_cluster` below, and some other directory (where the pre-generated certificates are) used for certificates_path?